packages/tools/guiEditor/src/components/propertyTab/propertyGrids/gui/commonControlPropertyGridComponent.tsx

```diff
@@ -93,6 +96,8 @@ export class CommonControlPropertyGridComponent extends React.Component<ICommonC
                       { label: "Brush Script MT", value: 9 },
                   ],
             value: 0,
+            invalidFonts: JSON.parse(String(window.sessionStorage.getItem("invalidFonts"))) ? JSON.parse(String(window.sessionStorage.getItem("invalidFonts"))) : [],

```

Comment:
You should do:
```typescript
invalidFonts: JSON.parse(String(window.sessionStorage.getItem("invalidFonts"))) ?? []
```
to avoid evaluating the expression two times.

Reply:
done!